vmdb/app/controllers/ems_common.rb

```diff
@@ -583,7 +583,7 @@ def set_form_vars
     @edit[:new][:hostname] = @ems.hostname
     @edit[:new][:ipaddress] = @ems.ipaddress
     @edit[:new][:emstype] = @ems.emstype
-    @edit[:amazon_regions] = get_amazon_regions if @ems.emstype == "ec2"
+    @edit[:amazon_regions] = get_amazon_regions if @ems.kind_of?(EmsAmazon)
```

Comment:
Is there a way to avoid `kind_of?`, maybe just asking `support_region`?

Not sure if we can make this generic. But as no surprise, I'm not a `kind_of` fan.